### ISSUE TITLE
Extension doesn't work with this JS bookmarklet

### ISSUE THREAD

**Issue by conuaunoc** [Vinne2/bookmark-iconizer#4]
I have this bookmarklet to convert a DuckDuckGo search to a Google search using DDG bang syntax (I'm using jQuery $ since it's loaded on the page):

```
javascript:(function(){if(!window.location.origin.includes('duckduckgo.com'))return;f=$('search_form');i=$('#search_form_input');i.val('!g '+i.val());f.submit()})()
```

Trying to add a favicon with this extension produces the following `bookmarks.html`

```
<!DOCTYPE NETSCAPE-Bookmark-file-1>
<!-- This is an automatically generated file.
     It will be read and overwritten.
     DO NOT EDIT! -->
<META HTTP-EQUIV="Content-Type" CONTENT="text/html; charset=UTF-8">
<TITLE>Bookmarks</TITLE>
<H1>Bookmarks</H1>
<DL><p>
	<DT><A HREF="javascript:(function(){if(!window.location.origin.includes('duckduckgo.com'))return;f=$('search_form');i=$('
```

Somewhere in the code, it's treating the `#` as a comment delimiter and ignoring everything after it.

**Comment by fharper** [comment]
Create one with a http link, update the HTML file and put the bookmarklet code inside the href before importing the bookmark.

**Comment by Kaligula0** [comment]
Fixed by dragonish in #5 (thanks!)

**Comment by Kaligula0** [comment]
(Version 1.7.3 is waiting for approval in Web Store).
